stable/oauth2-proxy/values.yaml

```diff
@@ -28,8 +40,6 @@ image:
   # - name: myRegistryKeySecretName
 
 extraArgs:
```

Comment:
Accoridng to the doc the default value is `{}`

Reply:
@Miouge1 you are right, so the documentation was incorrect before, what do you think we should do? Prefer to set it to `{}` or put the default args in the documentation?

I feel like the current value of `extraArgs` is a value we want to have even if the user override `extraArgs`, so I wonder if we shouldn't instead put this argument directly in the deployment and simply have `extraArgs` default to `{}`. WDYT?

Reply:
Considering that the `containerPort: 4180` is fixed in `templates/deployment.yaml`, I agree that it makes sense to set the `http-address` on top of the `extraArgs`

Reply:
@Miouge1 it's pushed :)